crates/node-builder/src/builder.rs

```diff
@@ -11,6 +11,7 @@ use crate::{
     rpc::{RethRpcServerHandles, RpcContext, RpcHooks},
     NodeHandle,
 };
+use alloy_primitives::utils::format_ether;
```

Comment:
I want to reexport this from reth-primitives instead

Reply:
SGTM, I'll adjust it later today